### PR TITLE
Ebook version 0.1, closes #449

### LINKED ISSUE #449: Ebook Version
It would be great if we can made a ebook version of the site. I've been wanting to read the entire guide for some time now, but I do not really like spend much time reading on the screen. I could try to convert the posts into a `.mobi` version using pandoc or this project: https://github.com/mccormicka/md-book

Currently I can send each individual page to my kindle using tinderizer, but a complete ebook of the guide would be great.

What do you think?

### PR DESCRIPTION
**Pull request by elbaulp**
- [X] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/gh-pages/CONTRIBUTING.md)?
- [X] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**
-----

This is a first attempt, the script could be improved.

### CODE REVIEW

**Comment by sofisofichafodze** [comment]
sofitchafodze  instagram code

File: script/generate-ebook [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#/bin/env bash
+
+# Author: #elbaulp
+
+mkdir tmp
+cd tmp
+cp ../../_articles/en-US/*.md .
+
+## Remove the front matter
```
Comment (MikeMcQuaid) [inline script/generate-ebook]: I wonder if Jekyll could be utilised to do this for us?

File: script/generate-ebook [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#/bin/env bash
```
Comment (MikeMcQuaid) [inline script/generate-ebook]: `#!/bin/bash` or `#!/usr/bin/env bash` would be better here. Would also be good to add a `set -e` so errors aren't silently ignored.

File: script/generate-ebook [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+# Author: #elbaulp
```
Comment (MikeMcQuaid) [inline script/generate-ebook]: Can skip this; the `git log` will reveal you are author.

File: script/generate-ebook [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+## Decrease sections by one level
+# perl -i -pe 's/^\#{2}/#/s' *.md
+
+# Correcting path for assets
+perl -i -pe 's/\/assets/\.\.\/\.\.\/assets/s' *.md
+
+# Generate title
+echo """
+---
+title: Opensource Guide
```
Comment (MikeMcQuaid) [inline script/generate-ebook]: `Open Source Guides`

File: script/generate-ebook [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+author: Github
```
Comment (MikeMcQuaid) [inline script/generate-ebook]: `GitHub and friends`

File: script/generate-ebook [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+rights: Attribution 4.0 International
```
Comment (MikeMcQuaid) [inline script/generate-ebook]: `Creative Commons Attribution 4.0 International`

**Comment by elbaulp** [comment]
Hi @MikeMcQuaid, I don't know Ruby, I could try in python, maybe.

**Comment by MikeMcQuaid** [comment]
@elbaulp Leaving it in Bash should be fine but I'd rather something like a Rubygem was installed or used in that case rather than manual `perl` substitutions. Thanks!

File: script/generate-ebook (hunk 2 above)
Comment (ockxdd) [reply script/generate-ebook]: https://guides.github.com/features/mastering-markdown/

File: script/generate-ebook (hunk 6 above)
Comment (ockxdd) [inline script/generate-ebook]: https://guides.github.com/features/mastering-markdown/

Comment (ockxdd) [reply script/generate-ebook]: #449

Comment (ockxdd) [inline script/generate-ebook]: #458

File: script/generate-ebook (hunk 2 above)
Comment (webknjaz) [reply script/generate-ebook]: I'd even suggest:
```bash
#! /usr/bin/env bash

if [ -n "$DEBUG" ]
then
  set -x
fi
set -euo pipefail
# ref: https://coderwall.com/p/fkfaqq/safer-bash-scripts-with-set-euxo-pipefail
```

**Comment by MikeMcQuaid** [comment]
Passing on this as there's been no movement in almost 2 months, sorry.

**Pull request closed without merging**